site/content/docs/5.1/components/orange-local-headers.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: docs
+title: Orange local headers
+description: Documentation and examples for Boosted's exclusive Brand responsive title bars, breadcrumbs and local navigation bars.
```

Comment:
Remove the reference to breadcrumbs and local navigation bars since it is not yet implemented.